tests/standalone/io/link_test.dart

```diff
@@ -239,7 +239,7 @@ testRenameSync() {
 
 void testLinkErrorSync() {
   Expect.throws(
-      () => new Link('some-dir-that-doent exist/some link file/bla/fisk')
+      () => new Link('some-dir-that-does-not exist/some link file/bla/fisk')
```

Comment:
I personally prefer `nonexistent`, but this seems like the simplest change matching the spirit of the test,

Reply:
Should be fine.